html/webappapis/dynamic-markup-insertion/document-write/module-delayed.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<title>async document.write in a module</title>
+<script src="/resources/tharness.js"></script>
```

Comment:
Should be `testharness.js`. I wonder if the lint should have caught it (cc @jgraham @gsnedders)

Reply:
We could have a lint for 'only these filenames in a /resources/ include', but I worry it would just annoy us when we added a new script.

For now, I've pushed a fixup to this CL.